blocklib/picoscope/test/qa_TimingMatcher.cc

```diff
@@ -535,6 +535,84 @@ const boost::ut::suite<"TimingMatchers"> TimingMatcherTests = [] {
             },
             result3.tags);
     };
+
+    "extended matching"_test = [&] {
+        using namespace boost::ut;
+        using namespace std::chrono_literals;
+
+        constexpr float    sampleRate   = 1e6f; // 1 µs/sample
+        constexpr unsigned nSamples     = 80;   //
+        constexpr auto     timeout      = 10us; // ⇒ maxDelaySamples = 10
+        unsigned long      acqTimestamp = 0;
+
+        TimingMatcher matcher{.timeout = timeout, .sampleRate = sampleRate};
+
+        // 1st match
+        std::vector<gr::property_map> tags1{                                //
+            generateTimingTag("EVT_A"s, acqTimestamp + 10'000, 0.0f, true), //
+            generateTimingTag("EVT_B"s, acqTimestamp + 25'000, 0.0f, true)};
```

Comment:
minor:
```suggestion
            generateTimingTag("EVT_A"s, acqTimestamp + 10'000, 0.0f, true),
            generateTimingTag("EVT_B"s, acqTimestamp + 25'000, 0.0f, true),
    };
```

Please use trailing comma instead of empty comments for multiline argument lists to correctly format with clang-format.